scripts/sct_utils.py

```diff
@@ -206,6 +206,99 @@ def run(cmd, verbose=1, error_exit='error', raise_exception=False):
         return status_output, output_final[0:-1]
 
 
+def check_exe(name):
```

Comment:
That code was BSD-licensed but since I'm the original author I'm OK with that. Careful next time!

Reply:
sorry, i though you suggested to use this piece of code in the comment you left in the issue. We can discuss the proper way of doing it in the next meeting.

Reply:
I suggested to use "something like it" but since it was my code, I have no problem with a verbatim copy. My point was more on the general case (if it weren't mine). But here everything is OK.